mdal/mdal_logger.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2020 Tomas Mizera (tomas.mizera2 at gmail dot com)
+*/
+
+#include <iostream>
+
+#include "mdal_logger.hpp"
+#include "mdal_utils.hpp"
+
+static MDAL_Status sLastStatus;
+static MDAL_LoggerCallback sLoggerCallback = MDAL::setDefaultLoggerCallback();
+
+void _log( MDAL_LogLevel logLevel, MDAL_Status status, std::string mssg )
+{
+  if ( sLoggerCallback )
```

Comment:
check geodiff, here you need to check the the MDAL_LogLevel < sMaximumLogLevel similar to geodiff. the maximum log level setup should be also in the mdal.h, so the user can select to see only errors or errors and warns and so on. by default only show errors.